ledger-service/http-json/src/failure/scala/http/FailureTests.scala

```diff
@@ -63,7 +63,7 @@ final class FailureTests
         (status, out) <- getRequestEncoded(uri.withPath(Uri.Path("/readyz")))
         _ = status shouldBe StatusCodes.ServiceUnavailable
         _ = out shouldBe
-          """[-] ledger failed
+          """[-] ledger failed (io.grpc.StatusRuntimeException: UNAVAILABLE: io exception)
```

Comment:
I'm not sure we should expose this level of detail, `UNAVAILABLE` and the exception message is probably enough, without the specific type, but we don't need to do it here.

Reply:
You’re assuming that we know that we can only get `StatusRuntimeException`s. I’m not sure that’s true. I think we can fail on lower levels of the stack as well.

Reply:
It's possible, but that's why I was suggesting to keep the message in. But it's not really important. :)